### ISSUE TITLE
Game rules change. Whether it is possible to change the way cards are selected

### ISSUE THREAD

**Issue by Yang011013** [lil-lab/cb2#6]
Hello, When I check the human-model Deployment data you provided, I find that when the agent wants to get the card in front, it inevitably passes the card under its feet, so the card under its feet is automatically selected, but it is not the card it wants to select. Could I change the rules so that standing on a card for a certain amount of time that the card is selected, or could I have a button that indicates that the card is selected when the button is tapped? If so, what part of the code should I change?

**Comment by jsharf** [comment]
Yes, you are free to do this. It's in src/cb2game/server/state.py. Look for CardSelection. Note that the logic includes both updating the game state and generating animation events -- you'll need to change both. Also, we wouldn't accept this change if you wanted to contribute upstream CB2, as it changes the rules of the game.

Unfortunately this part of the codebase is still a bit messy. Most of it has been cleaned up.

I recommend just letting users learn to "double step" on a card if they wish to pass without selecting -- doing the shuffle of stepped forward, backward, and then forward again.
